ImperatorToCK3/Data_Files/blankMod/output/localization/english/CONVERTER_bookmarks_l_english.yml

```
@@ -1,3 +1,3 @@
 ﻿l_english:
  bm_867_converted:0 "A different world"
- bm_867_converted_desc:1 "The world has changed."
\ No newline at end of file
+ bm_867_converted_desc:1 "Centuries have passed, since the day Romulus first laid the stones of Rome and Ulysses deviced the Wooden Horse. What has the world become until now? What will come next? Only the Gods know."
```

```suggestion
 bm_867_converted_desc:1 "Centuries have passed, since the day Romulus first laid the stones of Rome and Ulysses devised the Wooden Horse. What has the world become until now? What will come next? Only the Gods know.."
```
I'm not a native speaker, so